functions/typescript/modules/module2/index.complete.ts

```diff
@@ -30,7 +30,7 @@ const recordHandler = async (
   record: DynamoDBRecord,
   lambdaContext: Context
 ): Promise<void> => {
-  if (lambdaContext.getRemainingTimeInMillis() > 1000) {
+  if (lambdaContext.getRemainingTimeInMillis() < 1000) {
```

Comment:
I fixed this, ok?

Reply:
Yes, good catch - I need to fix it also in the workshop text.